src/test/scala/com/mozilla/telemetry/TestUtils.scala

```diff
@@ -86,6 +86,7 @@ object TestUtils {
     }
   }
 
+  // scalastyle:off methodLength
```

Comment:
I'm open to shortening this.

Reply:
I'll leave it like this for now. As far as I understand this part these util methods would lose a little on readability if refactored for shortening.